src/ch/imedias/rsccfx/view/RsccSupportPresenter.java

```diff
@@ -83,22 +78,21 @@ public String validationImage(String token) {
    */
   private void attachEvents() {
 
-    view.tokenTxt.setOnKeyPressed(event -> {
+    view.tokenTxt.setOnKeyReleased(event -> {
       view.isValidImg.setImage(new Image(validationImage(view.tokenTxt.getText())));
```

Comment:
I applied this, but i don't see what difference it makes. Maybe you can explain this sometimes. (after pulling the request :) )